docs/91_version_history.rst

```diff
@@ -35,229 +35,186 @@ Release Date: 2023-10-17
-       - A memory leak in one of the storage plugins has been fixed. This caused the storage service to consume large amounts of memory over time which could result in the operating system killing the service.
-       - The format of Python back traces has been improved to use multiple lines within the log. This makes the trace easier to understand and prevents the truncation that can occur.
-       - The setting of log levels fro a service is now also reflected in any Python code loaded by the service.
+       - Audit logs have been added to the user API of the public REST interface.
        - The plugin developers guide has been updated to include the mechanism for adding audit trail entries from C++ plugins.
        - Plugins that run within the south and north services and north tasks now have access to the audit logging system.
-       - The documentation regarding handling and updating certificates used for authentication has been updated. 
-       - A number of changes have been made to improve the performance of sending data north from the system.
-       - The performance of the statistics history task has been improved. It now makes fewer calls to the storage subsystem, improving the overall system performance.
-       - Changes have been made to the purge process in the sqlitelb and sqliteinmemory plugins in order to improve performance.
+       - The public API has been updated to include the ability to make control requests via the public API.
+       - The public API of the system has been updated to allow selection of readings from the storage buffer for given time intervals.
+       - New public API Entry Points have been added to allow for the creation and manipulation of control pipelines.
+       - The public API that is used to retrieve reading data from the storage layer has been updated to allow data for multiple assets to be retrieved in a single call.
+       - The ingestion of data from asynchronous south service paid no attention to the advanced configuration option "Throttle". This meant that very fast asynchronous south plugins could build extremely large queues of data within the south service, using system resources and taking a long time to shutdown. This has now been rectified, with asynchronous south services now subject to flow control if the "Throttle" option is set for the service. Unconstrained input is still available if the "Throttle" option is not checked.
        - The south plugin now supports three different modes of polling. Polling at fixed intervals from the time started, polling at fixed times or polling on demand via the control mechanisms.
+       - Support has been added to allow filters to ingest pass data onwards during a shutdown of the filter. This allows any buffered data to be flushed to the next filter in the pipeline.
+       - A security vulnerability in one of the PIP packages used by some of the Python components used by the system.
```

Comment:
Need a verb. Do you mean "A security vulnerability in one of the PIP packages used by some of the Python components _has been found_?"